lib/web_console/testing/fake_middleware.rb

```diff
@@ -0,0 +1,50 @@
+require "action_view"
+require "action_dispatch"
+require "web_console"
+
+module WebConsole::Testing
```

Comment:
I would prefer it written like:

```
module WebConsole
  module Testing
    class FakeMiddleware
    end
  end
end
```

That way, the file that is required first will create the `WebConsole::Testing` module and the other files will just reopen it.  You also get better `Module.nesting` and can reference constants in WebConsole relatively as well. Same for the precompiler.